movies/movies

```diff
@@ -170,7 +170,7 @@ getConfiguredPython || exit 1
 getConfiguredClient || exit 1
 checkInternet || exit 1 # check if we have a valid internet connection if this isnt true the rest of the script will not work so stop here
 
-while getopts 'ud:hv' flag; do
+while getopts 'udhv' flag; do
```

Comment:
This line still needs the colon after it for it to work. Like you had it before ```'ud:hv'```. Other than that this looks good to merge :+1: